packages/app/templates/ui-extensions/projects/checkout_ui/README.md

```diff
@@ -0,0 +1,87 @@
+
+## Tutorials
+---
+The best way to get started is to follow some of the available Extension Tutorials:
+
+
+For creating an extension:
+* [Enable Extended Delivery Instructions](https://shopify.dev/apps/checkout/delivery-instructions) - Static Renderer
+* [Adding Product Offer Recommendations](https://shopify.dev/apps/checkout/product-offers) - Static Renderer
+* [Creating a Custom Banner](https://shopify.dev/apps/checkout/custom-banners) - Dynamic Renderer
+
+To add specific features to an extension
+* [Adding Field Validation](https://shopify.dev/apps/checkout/validation)
+* [How to Localize an extension](https://shopify.dev/apps/checkout/localize-ui-extensions)
+
+
+## Getting Started
+---
+Initially, your extension will look like the following:
+
+```
+└── my-app
+  └── extensions
+    └── my-checkout-ui-extension
+        ├── src
+        │   └── index.jsx OR index.js // The index page of the checkout UI extension
+        ├── locales
+        │   ├── en.default.json // The default locale for the checkout UI extension
+        │   └── fr.json // The locale file for non-regional French translations
+        └── shopify.ui.extension.toml // The config file for the checkout UI extension
+
+```
+
+Start writing code for your extension simply editing the `src/index.js` file (or equivalent file extension of your choice).
+
+> By default, the Checkout UI Extension is preconfigured as a [`Dynamic Extension Point`](https://shopify.dev/api/checkout-extensions/checkout#extension-points).
+> You can change it to your desired Extension Point location.
+
+
+To shape your extension you have the following collection of tools available:
+* [UI Components](https://shopify.dev/api/checkout-extensions/checkout/components)
+* [Shopify APIs](https://shopify.dev/api/checkout-extensions/checkout/extension-points/api).
+
+With these tools you'll get access to Shopify-enabled resources (such as [Analytics](https://shopify.dev/api/checkout-extensions/checkout/extension-points/api#analytics), [Localization](https://shopify.dev/api/checkout-extensions/checkout/extension-points/api#localization) or [Storage](https://shopify.dev/api/checkout-extensions/checkout/extension-points/api#storage)) and the frameworks that you need to read (and mutate) data for your Checkout-UI extension (like Shop, Customer, CartLine, MetaFields, etc.).
+
+> If you are using React, there is also a large collection of [React Hooks available](https://shopify.dev/api/checkout-extensions/checkout/extension-points/api#react-hooks) to ease access to these operations, otherwise you'll need to manually subscribe to the subscribable value directly with a callback.
+
+## FAQ
```

Comment:
Another FAQ entry I think we want: "How do I let merchants customize my extension?", which should link to the guide on the merchant settings definition (https://shopify.dev/api/checkout-extensions/checkout/configuration#settings-definition) and the tutorial that shows an example of using them (https://shopify.dev/apps/checkout/custom-banners/add-custom-banner)